common/po/pt_BR.po

```diff
@@ -1724,6 +1724,14 @@ msgstr "AVISO"
 msgid "Exclude {path} from future snapshots?"
 msgstr "Excluir {path} de snapshots futuros?"
 
+#: qt/app.py:1189
+msgid "The following folders are missing"
+msgstr "As seguintes pastas estão faltando"
+
+#: qt/app.py:1189
+msgid "Do you want to proceed?"
+msgstr "Deseja prosseguir?"
+
```

Comment:
This will be overwritten. Never edit po-files directly.
Translations are done at our Weblate instance: https://translate.codeberg.org/projects/backintime/common/pt_BR/

For this PR you just need to mark the original source strings as translatable via using `_(...)`. No need to do more in this PR.

The synchronization between GitHub and Weblate (hosted at translate.codeberg.org) is not automatic for several reasons. The process is described in here: https://github.com/bit-team/backintime/blob/dev/common/doc-dev/2_localization.md . In short: I trigger the download of translations (po-fiels) from Weblate and in a second step I scan the py-files for updated/deleted/new source strings and integrate them into the po-files. In  a third step this po-files then uploaded again to Weblate.